docker-postgis/Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM postgres:10-alpine
```

Comment:
what is the relationship with this dockerfile and two dockerfiles in the root dir?

Reply:
DONE: simplified.